### PR TITLE
Add endpoints to call processes

### PR DESCRIPTION
**Pull request by RFlipper**
We added a few methods to our API

### CODE REVIEW

**Comment by RFlipper** [comment]
@sriyen-msft I added readme.md

File: certified-connectors/Plumsail Documents/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Plumsail Documents Connector
+Plumsail Documents provides a powerful and very extensive REST API.  Using this API, you can generate documents in Microsoft Flow, Azure Logic Apps or PowerApps.
```
Comment (sriyen-msft) [inline certified-connectors/Plumsail Documents/readme.md]: @RFlipper Please change the text from "Microsoft Flow" to "Microsoft Power Automate"

Comment (sriyen-msft) [reply certified-connectors/Plumsail Documents/readme.md]: @RFlipper Please change the text from "PowerApps" to "Power Apps" as well.

File: certified-connectors/Plumsail Documents/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+You will need the following to proceed:
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* A valid [Plumsail Account](https://account.plumsail.com)
+* The Power platform CLI tools
+
+### Deploying the sample
+Run the following commands and follow the prompts:
+
+```paconn
+paconn create --api-def apiDefinition.swagger.json --api-prop apiProperties.json --secret <client_secret>
```
Comment (dougelliott) [inline certified-connectors/Plumsail Documents/readme.md]: This specific command is for connectors that have OAuth has their auth method. It looks like yours uses API key, so this should instead be changed to:

`paconn create --api-prop [Path to apiProperties.json] --api-def [Path to apiDefinition.swagger.json] --icon [Path to icon.png]`

Comment (dougelliott) [reply certified-connectors/Plumsail Documents/readme.md]: Thanks for updating!

**Comment by sriyen-msft** [comment]
@RFlipper Please add back the x-ms-api-annotation to the info like below:

```
  "info": {
    "contact": {
      "email": "support@plumsail.com",
      "name": "Support",
      "url": "https://plumsail.com"
    },
    "description": "The API allows you to create documents from templates (DOCX or HTML file), convert it to PDF and many more. Visit https://plumsail.com/documents for more information",
    "title": "Plumsail Documents",
    "version": "v1",
    "x-ms-api-annotation": {
      "status": "Production"
    }
```

**Pull request merged**